src/gpu-compute/register_file_cache.cc

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2015-2017 Advanced Micro Devices, Inc.
```

Comment:
2023 / 2024 and probably UW Madison?

Reply:
Right -- I mentioned it in the other file but forgot to put "here too" here, thanks.